src/frontend/classic/components/MediaItemSelect.svelte

```diff
@@ -8,26 +8,28 @@
         ContextMenuGroup,
         ContextMenuOption,
         Loading,
-    } from "carbon-components-svelte";
-    import { EarthFilled } from "carbon-icons-svelte";
+    } from 'carbon-components-svelte';
+    import { EarthFilled } from 'carbon-icons-svelte';
 
-    import { fade } from "svelte/transition";
+    import { fade } from 'svelte/transition';
 
-    import MediaItem from "./MediaItem.svelte";
+    import MediaItem from './MediaItem.svelte';
     import { selectedMedia, selectedItem } from '../Stores';
 
-    import type { IMediaContainer } from "../../../engine/providers/MediaPlugin";
-    
+    import type { IMediaContainer } from '../../../engine/providers/MediaPlugin';
+    import { ItemMark, MarkType } from '../../../engine/ItemmarkManager';
 
     let items: IMediaContainer[] = [];
     let filteredItems: IMediaContainer[] = [];
+    let marks: ItemMark[] = [];
```

Comment:
Variable not really nedded

Reply:
marks is used as a cache to reduce the amount of Items.get on the manager ... but is that reallly needed ?
`mark={HakuNeko.ItemmarkManager.getMark(item, marks)}`
see comment : https://github.com/manga-download/haruneko/pull/51#discussion_r934261511

Reply:
If this is intended to be used as observable, consider finding a different approach to update UI